policy/watcher.go

```diff
@@ -24,7 +24,8 @@ const (
 	//hookToChain = "CATTLE_FORWARD"
 	hookToChain                  = "FORWARD"
 	cattleNetworkPolicyChainName = "CATTLE_NETWORK_POLICY"
-	ipsetNameMaxLength           = 31
+	cattleNetworkPolicyChainNameOut = "CATTLE_NETWORK_POLICY_OUT"
+	ipsetNameMaxLength           = 26
```

Comment:
That is strange because ipset gave me an error that 31 characters ARE too long.... I can not confirm what is right...